src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import core.basesyntax.suppliers.FigureSupplier;
+import java.util.Arrays;
+
+public class Main {
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] arrFigure = new Figure[6];
+        for (int i = 0; i < arrFigure.length; i++) {
+            if (i != arrFigure.length / 2) {
```

Comment:
Does your solution has expected behavior? I guess not.
```suggestion
            if (i < arrFigure.length / 2) {
```

Reply:
What's the difference?  🤷🏻‍♂️

Reply:
in your code if i = 5 the if statement will return true
we need to get elements that less then arr.length / 2 = 3 so need to get 0, 1, 2 elements from array